zeroeq/hbp/CMakeLists.txt

```diff
@@ -3,11 +3,7 @@
 # All rights reserved. Do not distribute without further notice.
 
 flatbuffers_generate_c_headers(HBP_FBS
-  detail/camera.fbs
```

Comment:
Keep this for the moment. It's needed by Neurscheme and I'm not removing it yet from RTNeuron.